explainer/index.bs

```diff
@@ -713,7 +713,75 @@ When using advanced methods to transfer data to the GPU (with a rolling list of
     </pre>
 </div>
 
-## Multi-Threading ## {#multi-threading}
+## Multithreading ## {#multithreading}
+
+Multithreading is a key part of modern graphics APIs.
+Unlike OpenGL, newer APIs allow applications to encode commands, submit work, transfer data to the GPU, and
+so on, from multiple threads at once, alleviating CPU bottlenecks.
+This is especially relevant to WebGPU, since IDL bindings are generally much slower than C calls.
+
+WebGPU does not *yet* allow multithreaded use of a single `GPUDevice`, but the API has been
+designed from the ground up with this in mind.
+This section describes the tentative plan for how it will work.
+
+As described in [[#gpu-process]], most WebGPU objects are actually just "handles" that refer to
+objects in the browser's GPU process.
+As such, it is relatively straightforward to allow these to be shared among threads.
+For example, a `GPUTexture` object can simply be `postMessage()`d to another thread, creating a
+new `GPUTexture` JavaScript object containing a handle to the *same* GPU process object.
+
+Several objects, like `GPUBuffer`, have client-side state.
+Applications still need to use them from multiple threads without `transfer`ring them back
```

Comment:
This is a bit unclear to me. Where are we controlling if something is transferred or not?

Reply:
Transferability is WebIDL `[Transferable]` and is controlled via the `transfer` list in [`postMessage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage).